### PR TITLE
PAAS-1292: generate vault credentials for 3.2.0 update script

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-1292

Short description:

### CODE REVIEW

**Review by laurentfufu — DISMISSED** [review]

File: jahia-cloud-env-update/update-to-3.2.0/update.yml [hunk 1]
```diff
@@ -146,3 +147,126 @@ actions:
...
+        role_id: ${response.role_id}
+        secret_id: ${response.secret_id}
+
+    - log: Check that credentials are actually working
+    - cmd[${nodes.bl.first.id}]: |-
+        curl -s -XPOST https://${settings.vault_hostname}/v1/auth/approle/login --data '{"secret_id": "${globals.secret_id}", "role_id": "${globals.role_id}"}' | jq -r .auth.policies | grep paas_environment
+
+    - if ( "${response.errOut}" != "" || "${response.out}" == "" ):
+        return:
+        type: success
```
Comment (adoux) [inline jahia-cloud-env-update/update-to-3.2.0/update.yml:243]: we really want to return a success type message?

File: jahia-cloud-env-update/update-to-3.2.0/update.yml [hunk 2]
```diff
@@ -146,3 +147,126 @@ actions:
...
+        message: "Generated vault credentials are not working"
+
+    - log: Store vault creds in en vars
```
Comment (adoux) [inline jahia-cloud-env-update/update-to-3.2.0/update.yml:246]: typo :grimacing:

**Review by adoux — CHANGES_REQUESTED** [review]

File: jahia-cloud-env-update/update-to-3.2.0/update.yml [hunk 3]
```diff
@@ -146,3 +147,126 @@ actions:
...
+    - log: Generate vault credentials
+    - cmd[proc]: |-
+        yum install -y jq > /dev/null
+
+        PAPI_HOSTNAME="${settings.papi_hostname}"
+        PAPI_TOKEN="${settings.papi_token}"
+        VAULT_HOSTNAME="${settings.vault_hostname}"
+        VAULT_TOKEN="${settings.vault_token}"
+        MAIL="${user.email}"
+        SHORTDOMAIN="paas_${env.shortdomain}"
```
[the remark this answers is not in the document]
Comment (Ronan-Gorain) [reply jahia-cloud-env-update/update-to-3.2.0/update.yml]: It seems my rebase was not really successful :)

File: jahia-cloud-env-update/update-to-3.2.0/update.yml (hunk 1 above)
Comment (Ronan-Gorain) [reply jahia-cloud-env-update/update-to-3.2.0/update.yml]: it's the way it works

File: jahia-cloud-env-update/update-to-3.2.0/update.yml (hunk 2 above)
Comment (Ronan-Gorain) [reply jahia-cloud-env-update/update-to-3.2.0/update.yml]: fixed

**Review by adoux — APPROVED** [review]

**Pull request merged**